File: library/src/main/java/com/snail/treeadapter/OnExpandableItemClickListerner.java
package com.snail.treeadapter;

import android.view.View;
import android.widget.AdapterView;

/**
 * 时间: 2017/12/5 11:12
 * 作者: 曾繁盛
 * 邮箱: 43068145@qq.com
 * 功能:
 */

public interface OnExpandableItemClickListerner<T extends Node<T>> {
    void onExpandableItemClick(T node, AdapterView<?> parent, View view, int position);
}

File: library/src/main/java/com/snail/treeadapter/OnExpandableItemClickListerner.kt
package com.snail.treeadapter

import android.view.View
import android.widget.AdapterView

/**
 * 时间: 2017/12/5 11:12
 * 作者: zengfansheng
 * 功能:
 */

interface OnExpandableItemClickListerner<T : Node<T>> {
    fun onExpandableItemClick(node: T, parent: AdapterView<*>, view: View, position: Int)
}
